src/cli/cli.cc

```diff
@@ -2022,7 +2022,7 @@ int main (const int argc, const char* argv[]) {
         settings["build_name"] + "_" +
         "v" + settings["meta_version"] + "-" +
         settings["meta_revision"] + "_" +
-        replace(platform.arch, "x86_64", "amd64")
+        replace(SSC::platform.arch, "x86_64", "amd64")
```

Comment:
just linking https://github.com/socketsupply/socket/issues/568 here, so we don't forget to fix it here later as well